docs/topics/native/native-objc-interop.md

```diff
@@ -43,11 +43,11 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a framewor
 >
 {style="warning"}
 
-To make your Kotlin code more Objective-C/Swift-friendly, you can hide a Kotlin declaration from Objective-C and Swift
+To make your Kotlin code more Swift/Objective-C-friendly, you can hide a Kotlin declaration from Objective-C and Swift
 with `@HiddenFromObjC`. The annotation disables a function or property export to Objective-C.
```

Comment:
To make your Kotlin code more Swift/Objective-C-friendly, use @HiddenFromObjC to hide a Kotlin declaration from Objective-C and Swift.

Reply:
WDYT?